source/deployment-options/wazuh-from-sources/wazuh-agent/index.rst

```diff
@@ -103,6 +103,12 @@ The Wazuh agent is a single and lightweight monitoring software. It is a multi-p
 
                         # pacman --noconfirm -Syu curl gcc make sudo wget expect gnupg perl-base perl fakeroot python brotli automake autoconf libtool gawk libsigsegv nodejs base-devel inetutils cmake
 
+                .. group-tab:: APK
+                    Alpine package keeper is a package manager that Alpine linux uses to manage its default packages.
+
+                    .. code-block:: console
+
+                        # apk --update add openssl openssl-dev libgcc make gcc g++ cmake curl automake linux-headers autoconf libtool procps python3 gettext-dev
 
```

Comment:
Before downloading and building Wazuh we need to include one more steps in order to avoid failures during the process.

To avoid this error:
```
rootcheck/os_string.c:71:10: fatal error: a.out.h: No such file or directory
   71 | #include <a.out.h>
      |          ^~~~~~~~~
```
We need to run:
```
ln -s /usr/include/linux/a.out.h /usr/include/a.out.h
```

Reply:
Does this work? Try to replace:
```c
#include <a.out.h>
```
With:
```c
#include <linux/a.out.h>
```

Reply:
I am trying with this change ! A prior it seems to work, I would like to test it a bit more and add it in the code.